src/main/java/org/fenixedu/academic/domain/accounting/PaymentCode.java

```diff
@@ -215,17 +215,12 @@ public final void process(Person responsiblePerson, SibsIncommingPaymentFileDeta
     }
 
 
-    public boolean matches (SibsIncommingPaymentFileDetailLine sibsDetailLine) {
-        final Person person = getPerson();
-        if (person == null) {
-            return false;
-        }
-
-        return person.getEventsSet()
+    public boolean matches(SibsIncommingPaymentFileDetailLine sibsDetailLine) {
```

Comment:
@sergiofbsilva This method no longer uses anything from the PaymentCode class. It should probably be moved somewhere else (eventually).